drivers/net/wireless/marvell/mwifiex/pcie_quirks.h

```diff
@@ -12,6 +12,7 @@
  */
 #define QUIRK_FW_RST_WSID_S3	BIT(1)
 #define QUIRK_NO_BRIDGE_D3	BIT(2)
+#define QUIRK_DO_FLR_ON_BRIDGE	BIT(4)
```

Comment:
Any reason for skipping `BIT(3)`?

Reply:
Whoops, I had another quirk for disabling ASPM L1.2 before that, which is obviously no longer needed now.

Reply:
Changed to `BIT(3)`